packages/syntax-highlighter/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
+    [
+      "@babel/env",
+      {
+        "targets": {
+          "browsers": ["last 2 versions"]
+        }
+      }
+    ],
+    "@babel/preset-react"
+  ],
+  "plugins": ["@babel/plugin-proposal-class-properties", "@babel/plugin-proposal-object-rest-spread"]
+}
```

Comment:
- [ ] pretty sure this file can be reverted?
- [ ] actually I'm pretty sure _all_ changes to the `syntax-highlighter` sub-package can be reverted!